package.json

```diff
@@ -15,6 +15,8 @@
     "build:server": "http-server dist",
     "dev": "nx run core:dev",
     "dev:server": "http-server build",
+    "pree2e": "npx playwright install",
```

Comment:
Do we still need this command?

Reply:
Otherwise, devs will always see browser packages are missing on the first run, and this command will come out on the terminal.

There is a way to avoid this by adding `playwright` to `package.json`, but it will increase the cache image size to 1.5G on CI and still need to run `npx playwright install` separately on CI. 😂

Reply:
1.5G 😱

Reply:
When run `npm run e2e`, `npm` will automatically execute `pree2e` before `e2e`.
On another thought, we can combine `pree2e` and `e2e` into one command. Does this look better?

Reply:
yes possibly better to combine them, less overhead.